packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -61,11 +61,17 @@ module.exports = function(env) {
 	let cliNodeModules = findAllNodeModules(__dirname);
 	let nodeModules = [...new Set([...userNodeModules, ...cliNodeModules])];
 
+	let compat = 'preact-compat';
+	try {
+		requireRelative.resolve('preact/compat', cwd);
```

Comment:
Should we be trying to resolve this only to `node_modules/preact/compat`? I think this will resolve to `node_modules/preact/compat/dist/compat.js`.